content/news/021/index.md

```diff
@@ -644,7 +644,7 @@ Rust discovery.
 
 ![How To Write a Crash Reporter](wor-crash-reporter.jpg)
 
-[@masonremaley][mason-remaley] wrote [an article](crash-reporter) walking
+[@masonremaley][mason-remaley] wrote [an article][crash-reporter] walking
```

Comment:
dammit, these typos in md link syntax are so hard to notice :(

Reply:
I miss them every time 😩